cmd/editorconfig/main.go

```diff
@@ -17,10 +17,24 @@ const (
 	Version = "1.2.0"
 )
 
+// Returns the absolute path of a file
+// if it is already absolute it returns
+// the given one
+func getAbsoluteFilePath(path string) string {
```

Comment:
Let's get rid of this function altogether

Reply:
Thanks, right, of course the function call isn't really needed and the code is also clear without it.